custodianship is a multi-value attribute, not a boolean

asaunier opened the pull request:
Fixes https://github.com/c2corg/v6_api/issues/207

tsauerwein commented on c2corg_api/scripts/migration/documents/waypoints/huts.py:
```diff
@@ -94,6 +95,16 @@ def get_document_locale(self, document_in, version):
             access_period=document_in.staffed_period
         )
 
+    def set_custodianship(self, document_in):
```
`get_custodianship`?

asaunier replied:
The goal of the function is to set the value of the ``custodianship`` attribute, isn't it?

tsauerwein replied:
You are using the return value of this function to set the attribute. But this function does not set an attribute.

asaunier replied:
Right. I have pushed a fixup.

tsauerwein commented on c2corg_api/models/waypoint.py:
```diff
@@ -139,7 +139,7 @@ class _WaypointMixin(object):
 
     # gardiennage (gite, camp_site, hut, abri, bivouac, base_camp)
-    custodianship = Column(Boolean)
+    custodianship = Column(enums.custodianship_type)
```
With 'multi-value' you don't mean you can select multiple values? You would have to use `Column(ArrayOfEnum(enums.custodianship_type))` in that case.

asaunier replied:
I meant it is a field with more value than yes/no. But only one value at a time can be selected. Do you think I should use another word than "multi-value"?

tsauerwein replied:
Ok. No, you don't have to change the commit message for me. I just wanted to make sure that we are talking about the same thing.

asaunier commented:
The spaces within the custodianship values seem to cause no problem as far as I can see.
But to be consistent with the other attributes I have added them. By the way I have replaced "stub" by "draft" in the quality types since the client prefers this term.

OK for merging the PR?

tsauerwein commented:
:+1:
